library/std/src/sys/windows/ext/io.rs

```diff
@@ -220,3 +220,141 @@ impl IntoRawSocket for net::UdpSocket {
         self.into_inner().into_socket().into_inner()
     }
 }
+
+#[unstable(feature = "is_terminal", issue = "80937")]
+impl io::IsTerminal for sys::stdio::Stdin {
+    fn is_terminal() -> bool {
+        use c::{
+            STD_ERROR_HANDLE as STD_ERROR, STD_INPUT_HANDLE as STD_INPUT,
+            STD_OUTPUT_HANDLE as STD_OUTPUT,
+        };
```

Comment:
Is it a possibility to just remove these 3 `use`s and use them directly like `c::STD_ERROR_HANDLE` etc. like it's done in other parts of this file? See e.g. https://github.com/rust-lang/rust/blob/9213d20487b9f5d3fb538cb16b95fd8118b024dc/library/std/src/sys/windows/ext/io.rs#L70